app/src/main/java/com/teamfillin/fillin/presentation/AddPhotoActivity.kt

```diff
@@ -0,0 +1,29 @@
+package com.teamfillin.fillin.presentation
+
+import android.os.Bundle
+import com.teamfillin.fillin.R
+import com.teamfillin.fillin.core.base.BindingActivity
+import com.teamfillin.fillin.databinding.ActivityAddPhotoBinding
+
+class AddPhotoActivity : BindingActivity<ActivityAddPhotoBinding>(R.layout.activity_add_photo) {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding.btnAddPhoto.setOnClickListener {
+            val addPhotoDialog=CustomAddCompleteDialog(this)
+            addPhotoDialog.showDialog()
+        }
+
+        binding.btnBack.setOnClickListener {
+            val addCancelDialog=CustomAddCancelDialog(this)
+            addCancelDialog.showDialog()
+            addCancelDialog.setOnClickListener(object:CustomAddCancelDialog.OnDialogClickListener{
+                override fun onClicked() {
+                    //액티비티 나가기
+                }
+            })
```

Comment:
```suggestion
            addCancelDialog.setOnClickListener(CustomAddCancelDialog.OnDialogClickListener {
                // 이런식으로 작성 가능!
            })
```

아래와 같이 fun interface로 바꾸면 이렇게 바꾸라고 뜰거임!